facebook/jest
# Fix docs links with new language scope

**Summary**

Following https://github.com/facebook/jest/pull/4670 and https://github.com/facebook/jest/issues/4663
It fixes all links to documentation with new language scope.

**Test plan**

- Click on changed links.
- Create a `jest.config.js` file in a dummy project with a wrong preset, you should get a console error with the proper links

```
● Validation Error:

  Preset es2017 not found.

  Configuration Documentation:
  https://facebook.github.io/jest/docs/en/configuration.html
```
